docs/integrations/uipath.mdx

```diff
@@ -0,0 +1,125 @@
+---
+sidebar_position: 1
+title: UiPath
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+# Unlock Document Parsing with Cradl AI in UiPath
+
+<br/>
+
+This guide will show you how to extract data from your documents with Cradl AI in UiPath.
+By the end of this guide you should be able to:
+ - Train your own AI model customized to your own documents.
+ - Set up a feedback loop so your AI model can train and get better for every document you process.
+ - Process your documents in your UiPath workflow using Cradl AI.
+
+All you need to get started is:
+- UiPath studio installed on your computer.
+- Account on Cradl AI (If you don't have one yet, you can sign up [here](https://app.cradl.ai/login?redirect=signup&utm_campaign=docs)).
+
+:::tip Scalable
+Cradl AI empowers you with the ability to create your own AI models. 15 examples are all you need to get started.
+:::
+
+<br/>
+
+## Overview
+
+<br/>
+The Cradl AI integration for UiPath can be downloaded [here](https://www.nuget.org/packages/CradlAI.Activities), and consists of these activities:
+
+| Name        | Description |
+|:------------| :-----------|
+| Parse Document                        | Parse a document with a model in Cradl AI. |
+| Parse Document With Human In The Loop | Parse a document with a flow in Cradl AI. |
+| List Processed Documents              | Retrieve data from a flow in Cradl AI. |
+| Mark Run As Completed                 | Mark a Run as completed to avoid retrieving the same document twice. |
```

Comment:
Two things here: 
1. The other three activities' descriptions don't explain the reason for why the activity exists
2. "Run" is written capitalized in this description, yet "flow", "model", "document" are lower-case. Should be consistent.
```suggestion
| Mark Run As Completed                 | Set a run's status to "completed" |
```